ufw/config/file.sls

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.package.install' %}
+{%- from tplroot ~ "/map.jinja" import ufw with context %}
+{%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
+
+include:
+  - {{ sls_package_install }}
+
+ufw-default-file-file-managed:
+  file.managed:
+    - name:     {{ ufw.default_file }}
+    - user:     root
+    - group:    root
+    - template: jinja
+    - source: {{ files_switch(['ufw.default.tmpl', 'ufw.default.tmpl.jinja'],
+                              lookup='ufw-default-file-file-managed'
+                 )
+              }}
+    - require:
+      - sls: {{ sls_package_install }}
+    - context:
+        config: {{ ufw.settings | json }}
```

Comment:
I would strongly recommend avoiding using `config` here, since it leads to using `config.get`, which can easily be confused with the [`config.get`](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.config.html#salt.modules.config.get).

Use anything else, such as `ufw_config`, `ufw_settings`, etc.